packages/gatsby-telemetry/src/postinstall.js

```diff
@@ -0,0 +1,5 @@
+console.log(
```

Comment:
This is going to run even if the user has already opted out.

Also, this will run every time the user installs Gatsby — even a normal `npm install gatsby` to upgrade — which seems noisy/excessive.

Perhaps we just include a global counter and show the notice a few times and then stop?

Reply:
@KyleAMathews great point, do we actually want to log this when installing, or should we only log this on the first time `gatsby` command is run?

Reply:
@KyleAMathews I have now updated this, but I'm not sure if checking and updating the config is the approach we want to take, especially I'm uncertain about updating the config here.

Reply:
yeah, I think just showing when installing the `gatsby-cli` is probably enough

Reply:
We don't show any other postinstall messages there so it should stand out enough to be read.